specification/app/Microsoft.App/models/code-execution.tsp

```diff
@@ -0,0 +1,84 @@
+import "./common.tsp";
+using TypeSpec.Http;
+using TypeSpec.Rest;
+
+namespace Microsoft.App;
+
+@doc("The request to execute code.")
+model CodeExecutionRequest {
```

Comment:
`CodeInterpreterCodeExecutionRequest` ? 
Similarly file name change and description. 

Why ? 
This will help reading API spec  easily

Reply:
Was planning on changing this to CodeInterpreterSessionCodeExecutionRequest same as the file apis, but seems too long, so I chose to use SessionCodeExecutionRequest and align everything for now.